Sources/SwiftSyntaxBuilder/BinaryOperatorExprConvenienceInitializers.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+extension BinaryOperatorExpr {
+  public init(_ text: String) {
+    self.init(operatorToken: TokenSyntax.spacedBinaryOperator(text))
+  }
+}
+
+extension BinaryOperatorExpr: ExpressibleByStringLiteral {
```

Comment:
Do we actually need the `ExpressibleByStringLiteral` conformance? IIUC we should be able to use `String`s for binary operators in all relevant places because we have `String: ExpressibleAsBinaryOperatorExpr`.

Reply:
It's possible yes.